Documentation/DataProcessing/_RecordTransformationProcessor/_WithDatabaseFluidTemplate.typoscript

```diff
@@ -0,0 +1,10 @@
+# tt_content.text = FLUIDTEMPLATE
```

Comment:
Why it is commented?

Reply:
Because we don't set it here, it is just information what kind of cobject we are in